portfolio/src/main/java/com/google/sps/Event.java

```diff
@@ -0,0 +1,96 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Event is the container class for when a specific group of people are meeting and are therefore
+ * busy. Events are considered read-only.
+ */
+public final class Event {
+  private final String title;
+  private final TimeRange when;
+  private final Set<String> attendees = new HashSet<>();
+
+  /**
+   * Creates a new event.
+   *
+   * @param title The human-readable name for the event. Must be non-null.
+   * @param when The time when the event takes place. Must be non-null.
+   * @param attendees The collection of people attending the event. Must be non-null.
+   */
+  public Event(String title, TimeRange when, Collection<String> attendees) {
```

Comment:
Bit late for this, but bonus points if you add a Builder for this class. :P

(Not really necessary in this case, this'd just be as an exercise.)